product/reports/310_Migration Readiness - Providers/010_VMware Environment Summary.yaml

```diff
@@ -14,9 +14,11 @@ include:
     columns:
     - name
     - api_version
+    - id 
```

Comment:
@mturley Can you fix the trailing spaces here?

Reply:
Done, sorry about that. Thought I had my editor set to flag those.. 🤔